### DISCUSSION TITLE
Deleting archives doesn't delete the files on disk

### DISCUSSION THREAD

**Discussion by CodeOwlTree** [Difegue/LANraragi#683] [Q&A] [answered]
**LRR Version and OS**
Docker ("latest", so I think version 0.8.6) running locally on an Arch Linux PC.

**Bug Details**
I've noticed that deleting archives doesn't delete the actual files on disk. The comics don't appear in Lanraragi, but they remain on disk. I checked both the local volume and inside the running docker container with ```docker exec -it lanraragi sh```. Files from deleted archives remain in both places. My *docker-compose.yaml*:

```
# docker-compose.yaml
services:
  lanraragi:
    image: 'difegue/lanraragi:latest'
    container_name: 'lanraragi'
    ports:
      - '3000:3000'
    volumes:
      - '/home/my_user/local/lanraragi/content:/home/koyomi/lanraragi/content'
      - '/home/my_user/local/lanraragi/database:/home/koyomi/lanraragi/database'
```

**Comment by Difegue** [comment, marked as answer]
👋 This sounds like a permission problem -- Can the LRR container user modify files in your content folder?  

Your compose file doesn't seem to have `LRR_UID/LRR_GID`, so I suspect it's using the default 9001/9001 user in this case.  
See https://sugoi.gitbook.io/lanraragi/v/dev/installing-lanraragi/docker#changing-the-user-id-in-case-of-permission-issues .

**Reply by CodeOwlTree** [reply]
Thanks for the reply. That was basically the issue. Sorry for filing it as a bug report.

I wanted to still be able to startup LANraragi with ``docker-compose up`` without passing environmental variables as command line arguments. So here is what I did:

```
# .env
# Define's ID:GID for container
LRR_UID='1000' # Set user id to 1000
LRR_GID='1000' # Set group id to 1000
```
```
# docker-compose.yaml
services:
  lanraragi:
    environment:
      - LRR_UID=${LRR_UID} # Defined in .env
      - LRR_GID=${LRR_GID} # Defined in .env
    image: 'difegue/lanraragi:latest'
    container_name: 'lanraragi'
    ports:
      - '3000:3000'
    volumes:
      # Changed to relative path
      - '/home/my_user/local/lanraragi/content:/home/koyomi/lanraragi/content'
      - '/home/my_user/local/lanraragi/database:/home/koyomi/lanraragi/database'
```
I just type ```docker-compose up``` and everything works. I can delete archives files within LANraragi. I can even use LANraragi to import files from outside of volume folders.
